src/lib/components/Head/Head.svelte

```diff
@@ -7,63 +7,64 @@
 		/** the HTML attributes to attach to the meta tag */
 		attributes?: Record<string, string> | null | undefined;
 	}
-
```

Comment:
can you please keep the newlines? thanks!

Reply:
Oh sorry! Weird, I thought that's something Prettier automatically does. TIL!